setup.py

```diff
@@ -40,7 +40,7 @@ def find_version(*paths):
         "pandas == 1.3.5",
         "pyarrow == 10.0.1",
         "pydantic == 1.10.4",
-        "ray[default] == 2.0.0",
+        "ray[default] == 2.2.0",
```

Comment:
Since we will still want to also run tests using Ray 2.0.0 (i.e. via the AWS Glue on Ray preview), can we change this to `>= 2.0.0`?
```suggestion
        "ray[default] >= 2.0.0",
```
We should also propagate the same change over to `requirements.txt`.